powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/InitActivationRequest.java

```diff
@@ -37,18 +39,38 @@
 public class InitActivationRequest {
 
     private ActivationProtocol protocol = ActivationProtocol.POWERAUTH;
+
+    @Schema(description = "The identifier of the user")
+    @NotBlank(message = "User ID must not be empty when initiating activation")
     private String userId;
+
+    @Schema(description = "The identifier of the application")
+    @NotNull(message = "Application ID must not be null when initiating activation")
     private String applicationId;
+
+    @Schema(description = "Timestamp of activation expiration")
+    @Future(message = "The activation expiration timestamp must be in the future when initiating activation")
     private Date timestampActivationExpire;
+
+    @Schema(description = "Maximum number of failures for the activation")
+    @Positive(message = "Maximum failure count must be positive when initiating activation")
     private Long maxFailureCount;
+
     /**
      * @deprecated use {@link #activationOtp} for enabling OTP check and {@link #commitPhase} for controlling activation commit
      */
+    @Schema(description = "Activation OTP validation (deprecated)", deprecated = true)
```

Comment:
Suggestion to make it more clear.
```suggestion
    @Schema(description = "Activation OTP validation strategy (deprecated)", deprecated = true)
```